Orchestrator/docs/BFOrchestratorReport.md

```diff
@@ -1,109 +1,146 @@
+
+Use the following guidance to interpret the report.
+
+
+
+# Report Organization
+
+The test command thus produces a folder with HTML report and a few supporting artifacts as follows:
+
+- orchestrator_testing_set_ground_truth_instances.json: **TBD**
+- orchestrator_testing_set_labels.txt: **TBD**
+- orchestrator_testing_set_prediction_instances.json: **TBD**
+- orchestrator_testing_set_scores.txt: **TBD**
+- orchestrator_testing_set_summary.html: Report summary in HTML format
+
+The report summary contains several sections as follows:
+
+## Intent / Utterance Statistics  
+
+This section contains descriptive statistics **TBD: Bot audience is not familiar with term descriptive statistics. Use simpler language** of labels and utterances.
+
 It has two statistical sections, one for labels, the other utterances:
 
 - Label statistics
 - Utterance statistics
 
 ### Label statistics
 
-Label statistics lists the number of utterances labeled to each label.
-Additional metrics include utterance prevalence (ratio) for every label.
-The distributions can give Orchestrator users an overall view of the labels and utterances,
-and whether the distributions are skewed and emphasize too much on some labels, but not others.
+Label statistics lists the number of utterances labeled to each label. Additional metrics include utterance prevalence (ratio) for every label. The distributions can give Orchestrator users an overall view of the labels and utterances, and whether the distributions are skewed and emphasize too much on some labels, but not others.
 
 ### Utterance statistics
 
-On the other hand, utterance statistics focus on the #label distribution by the utterances.
-Some utterances are labeled with more than one intents, which might not be desirable.
-This table reflects the distribution of multi-label utterances.
+On the other hand, utterance statistics focus on the #label distribution by the utterances. Some utterances are labeled with more than one intents, which might not be desirable. This table reflects the distribution of multi-label utterances.
+
+### How to use this section
+
+**TBD**
+
+
 
 ## Utterance Duplicates
 
-In this section, the report lists the utterances tagged with more than one labels.
-Sometimes some dataset might contain utterances tagged with the same labels multiple times.
+This section reports on utterances with duplicate or multiple labels. A duplicate utterance is detected when it is present more than once. Thus, the report lists the utterances tagged with more than one labels. Sometimes some dataset might contain utterances tagged with the same labels multiple times.
+
 The report also lists the redundancy.
 
 - Multi-label utterances and their labels
 - Duplicate utterance and label pairs
 
+### How to use this section

```

Comment:
Do we need this TBD?  Just remove and add it later?

Reply:
This is the placeholder for Hung-chih. Once I'm done with these files, Hung-chih will work on those.